packages/collector/test/tracing/opentelemetry/test.js

```diff
@@ -516,7 +520,8 @@ mochaSuiteFn('opentelemetry/instrumentations', function () {
           }
         });
 
-        await controls.startAndWaitForAgentConnection();
+        // Waiting time added for the agent connection
```

Comment:
Suggestion: This comment describes "what" you have "done" while coding. IMO doesn't add any benefit for the team when working again on this code.

```suggestion
```

Reply:
Alternatively you could say: We need to increase the waiting timeout here for the initial azure connection, because it can take up to 1-2 minutes til Azure replies.